### ISSUE TITLE
Restore artifacts moved to maven.scijava.org?

### ISSUE THREAD

**Issue by chris-allan** [saalfeldlab/maven#3]
Recently we had a user raise an an issue (glencoesoftware/bioformats2raw#44) building `bioformats2raw`. This boiled down to the fact that in 388a19adc676f100b609d656c22ffd3eb737f8ec most of the N5 artifacts were removed from this repository and consequently disappeared from `https://saalfeldlab.github.io/maven/`.

I know that this happened almost a month ago now but this has had the rather unfortunate result of breaking current builds of downstream projects that have not updated their repository definitions as well as the builds of historical tags.

We certainly support the move to maven.scijava.org and its future position as the authoritative location for N5 artifacts but would you consider restoring them in this repository?

/cc @ctrueden, @joshmoore, @melissalinkert

**Comment by chris-allan** [comment]
Bump. Thoughts, @axtimwalde, @igorpisarev?

**Comment by ctrueden** [comment]
Restoring the previously deployed artifacts here seems reasonable to me—I don't see how it would cause any harm. As long as the binaries match what's on maven.scijava.org, of course. What do you think, @axtimwalde?

**Comment by axtimwalde** [comment]
This repository still exists with all artifacts that were here before, please check: https://saalfeldlab.github.io/maven/org/janelia/saalfeldlab/n5-zarr/maven-metadata.xml
I haven't added the most recent artifacts for all projects that have moved to maven.scijava.org.  Is that the issue?  I am hesitant to duplicate them here because one of the reasons to go scijava.org was that distribution is easier.

**Comment by ctrueden** [comment]
@chris-allan Can you post the build log from one of the failed builds? Maybe something else is going on...

**Comment by axtimwalde** [comment]
In the original issue we can see that they were trying to pull n5-zarr-0.0.4 which was never released to this repository.  So some BOM (pom-scijava?) was updated or the version number bumped without also adding the maven.scijava.org repository.

**Comment by ctrueden** [comment]
Ah, gotcha. Yeah, I think the right resolution is for the downstream project to fix their POMs, then.

**Comment by chris-allan** [comment]
Thanks for taking a look @axtimwalde, @ctrueden. Apologies for wasting people's time.

I think the source of the confusion here is me going too fast and not thoroughly reading all the commits. I made a poor assumption from reading the commit message that 388a19adc676f100b609d656c22ffd3eb737f8ec removed the JARs from `https://saalfeldlab.github.io/maven/`. I'm sure it didn't help at the time that `https://saalfeldlab.github.io/maven/org/janelia/saalfeldlab/n5-zarr/` for example returns 404.

The actual core software issue is that for testing `raw2ometiff` was consuming `SNAPSHOT` versions of `bioformats2raw` whose dependencies were updated to n5-zarr 0.0.4. `raw2ometiff` itself was still on 0.0.3-SNAPSHOT versions we built and deployed ourselves which included saalfeldlab/n5-zarr#2 and of course consequently has not had the`maven.scijava.org` change. Thus, we have dependencies being included without the requisite repositories being on the search path.
